types/dot-object/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for Dot-Object v1.5
+// Type definitions for Dot-Object 1.7
 // Project: https://github.com/rhalff/dot-object
 // Definitions by: Niko Kovačič <https://github.com/nkovacic>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
```

Comment:
Please add `// TypeScript Version: 2.2` on the following line as suggested by the CI.

Reply:
Thanks for the suggestion. It actually goes at the end of the comments block (for future reference), but your suggestion sent me on the correct path 👍